06-blockchain/03-alphabet.md

```diff
@@ -18,68 +20,65 @@ Alphabet contract does not produce notifications to process\.
 func Emit()
 ```
 
-Emit method produces sidechain GAS and distributes it among Inner Ring nodes and proxy contract\. It can be invoked only by an Alphabet node of the Inner Ring\.
+Emit method produces sidechain GAS and distributes it among Inner Ring nodes and proxy contract. It can be invoked only by an Alphabet node of the Inner Ring.
 
-To produce GAS\, an alphabet contract transfers all available NEO from the contract account to itself\. If notary is enabled\, 50% of the GAS in the contract account are transferred to proxy contract\. 43\.75% of the GAS are equally distributed among all Inner Ring nodes\. Remaining 6\.25% of the GAS stay in the contract\.
-
-If notary is disabled\, 87\.5% of the GAS are equally distributed among all Inner Ring nodes\. Remaining 12\.5% of the GAS stay in the contract\.
+To produce GAS, an alphabet contract transfers all available NEO from the contract account to itself. 50% of the GAS in the contract account are transferred to proxy contract. 43.75% of the GAS are equally distributed among all Inner Ring nodes. Remaining 6.25% of the GAS stay in the contract.
 
 ##### Gas
 
 ```go
 func Gas() int
 ```
 
-GAS returns the amount of the sidechain GAS stored in the contract account\.
+Gas returns the amount of the sidechain GAS stored in the contract account.
```

Comment:
relates to "update to [...] contract API"?

Reply:
New `gomarkdoc`.